src/main.rs

```diff
@@ -102,8 +103,10 @@ fn build_response_json(people_present: Option<u32>, raspi_temperature: Option<f3
 }
 
 fn status_endpoint(_: Request, res: Response<Fresh>) {
-    let mut res = res.start().unwrap();
+    // Move response into mutable variable
+    let mut res = res;
```

Comment:
D'oh of course. I first tried `res: mut Response<Fresh>` which didn't work, then moved to the variable moving.

I'll fix this in another commit.